src/data/equiv/mul_add.lean

```diff
@@ -545,6 +567,9 @@ protected def mul_left₀ (a : G) (ha : a ≠ 0) : perm G :=
   left_inv := λ x, by { dsimp, rw [← mul_assoc, inv_mul_cancel ha, one_mul] },
   right_inv := λ x, by { dsimp, rw [← mul_assoc, mul_inv_cancel ha, one_mul] } }
 
+lemma mul_left_bijective₀ (a : G) (ha : a ≠ 0) : function.bijective ((*) a : G → G) :=
```

Comment:
We have something of this name already:

https://github.com/leanprover-community/mathlib/blob/c5456607cc0415230c4ad0585c4300ffe36739bd/src/ring_theory/integral_domain.lean#L39-L40

so maybe that one needs `fintype` in the name.

Reply:
Hm well those lemmas should be for a `cancel_monoid_with_zero` instead it seems so I'll move them and rename them a bit

Reply:
Well I couldn't find a place for them, so I left them where they are but generalized a little